.travis.yml

```diff
@@ -30,6 +31,8 @@ install:
   - pushd geopyspark-backend &&
     ./sbt "project geotrellis-backend" assembly &&
     cp geotrellis/target/scala-2.11/geotrellis-backend-assembly-0.3.0.jar ../geopyspark/jars &&
+    ./sbt "project vectorpipe" assembly &&
+    cp vectorpipe/target/scala-2.11/vectorpipe-assembly-0.3.0.jar ../geopyspark/jars &&
```

Comment:
Could the version be either and ENV var or wildcard so this doesn't need to be updated for every pump?

Reply:
Yeah, I could implement something like that to make things easier.